.github/workflows/node.js.yaml

```diff
@@ -0,0 +1,167 @@
+name: Node.js CI
```

Comment:
Let's rename this to "CI" or "Continuous Integration" and the file to `ci.yml`.  The node.js part is irrelevant.